tools/webpack/vendors.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+const { join } = require( 'path' );
+
+const baseDir = join( __dirname, '../../' );
+
+module.exports = function( env = { environment: 'production', watch: false } ) {
+	const mode = env.environment;
+	const suffix = mode === 'production' ? '.min': '';
+
+	const vendors = {
+		'lodash': 'lodash',
+		'wp-polyfill': '@babel/polyfill',
```

Comment:
This one is more than that actually but it's mostly handled via add_inline_script to avoid loading more than what we really need, but it also means we need to include here all these dependencies (fetch polyfill, element-closest, form-data, node-contains. We also need TinyMCE list plugin (not sure if this one is already in Core though)

Reply:
I've added the other polyfills. Core has the TinyMCE lists plugin, what was the reason that Gutenberg included it? Just to get the latest version?

Reply:
Looks like core already has the required version of TinyMCE: https://core.trac.wordpress.org/changeset/43472. Even in the 5.0 branch.